README.md

```diff
@@ -1 +1,16 @@
 # University app
+
+### Run bundler:
+```gem install bundler```
```

Comment:
Maybe would be better to provide information about how to use docker. What do you think?